templates/cluster-template-system-assigned-identity.yaml

```diff
@@ -25,6 +25,10 @@ metadata:
   name: ${CLUSTER_NAME}
   namespace: default
 spec:
+  identityRef:
```

Comment:
while you're here, could you please also add it to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/templates/flavors/user-assigned-identity/kustomization.yaml which has the same issue?

Reply:
done